hack/aws-image-job.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: aws-image-creation
+  namespace: openshift-sandboxed-containers-operator
+spec:
+  parallelism: 1
+  completions: 1
+  backoffLimit: 1
+  template:
+    metadata:
+      name: aws-image-creation
+# needs hostnet for ocp
+    spec:
+      volumes:
+      - name: shared-data
+        emptyDir: {}
+
+      hostNetwork: true # needed for imds access
```

Comment:
How about we comment `hostNetwork` and retrieve the required config variables from secret like you are doing for Azure? This will keep it consistent as well as use the least required privileges